introductory/src/App.jsx

```diff
@@ -1,9 +1,21 @@
+import React from "react";
```

Comment:
You no longer need to import React from "react". Starting from the release 17 of React, JSX is automatically transformed without using React.createElement.
However, other exports like hooks must be imported.